docs-ref-services/key-vault.md

```diff
@@ -31,57 +30,55 @@ Use the management library to:
+pip install azure-identity
 ```
 
 ## Examples
 
-The following examples use service principal authentication, which is the recommended sign in method
-for applications that connect to Azure. To learn about service principal authentication, see
-[Authenticate with the Azure SDK for Python](https://docs.microsoft.com/en-us/python/azure/python-sdk-azure-authenticate)
-
 Retrieve the public portion of an asymmetric key from a vault:
 
 ```python
-from azure.keyvault import KeyVaultClient
-from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.keys import KeyClient
 
-credentials = ServicePrincipalCredentials(
-    client_id = '...',
-    secret = '...',
-    tenant = '...'
-)
+credential = DefaultAzureCredential()
 
-client = KeyVaultClient(credentials)
+key_client = KeyClient("https://<vaultname>.vault.azure.net", credential)
 
-# VAULT_URL must be in the format 'https://<vaultname>.vault.azure.net'
-# KEY_VERSION is required, and can be obtained with the KeyVaultClient.get_key_versions(self, vault_url, key_name) API
-key_bundle = client.get_key(VAULT_URL, KEY_NAME, KEY_VERSION)
-key = key_bundle.key
+//NOTE: please replace the ("<your-key-name>") with the name of your key in the vault 
+key = key_client.get_key("<your-key-name>")
+print(key.name)
 ```
 
 Retrieve a secret from a vault:
 
 ```python
-from azure.keyvault import KeyVaultClient
-from azure.common.credentials import ServicePrincipalCredentials
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
 
-credentials = ServicePrincipalCredentials(
-    client_id = '...',
-    secret = '...',
-    tenant = '...'
-)
+credential = DefaultAzureCredential()
 
-client = KeyVaultClient(credentials)
+secret_client = SecretClient(vault_url="https://<vaultname>.vault.azure.net", credential=credential)
+
+//NOTE: please replace the ("<your-secret-name>") with the name of the secret in your vault
+secret = secret_client.get_secret("secret-name")
+
+print(secret.name)
+print(secret.value)
 
-# VAULT_URL must be in the format 'https://<vaultname>.vault.azure.net'
-# SECRET_VERSION is required, and can be obtained with the KeyVaultClient.get_secret_versions(self, vault_url, secret_id) API
-secret_bundle = client.get_secret(VAULT_URL, SECRET_ID, SECRET_VERSION)
-secret = secret_bundle.value
 ```
 
+| Service | Package | README | Samples | API Reference | Changelog |
+| ------- | ------- | ------ | ------- | ------------- | --------- |
+| Identity | [azure-identity - 1.0.0](https://pypi.org/project/azure-identity/1.0.0) | [README](https://github.com/Azure/azure-sdk-for-python/blob/azure-identity_1.0.0/sdk/identity/azure-identity/README.md) | N/A | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-python/blob/azure-identity_1.0.0/sdk/identity/azure-identity/HISTORY.md) |
```

Comment:
This would need an update. Latest non-beta packages are azure-identity 1.3.0, azure-keyvault-* 4.1.0